udp_mux_universal.go

```diff
@@ -85,7 +85,8 @@ func (m *UniversalUDPMuxDefault) GetRelayedAddr(turnAddr net.Addr, deadline time
 // GetConnForURL add uniques to the muxed connection by concatenating ufrag and URL (e.g. STUN URL) to be able to support multiple STUN/TURN servers
 // and return a unique connection per server.
 func (m *UniversalUDPMuxDefault) GetConnForURL(ufrag string, url string) (net.PacketConn, error) {
-	return m.UDPMuxDefault.GetConn(fmt.Sprintf("%s%s", ufrag, url))
+	// TODO: Implement IP version detection
```

Comment:
The linter is complaining about this `TODO`.

How should we handle this? Should we break the public API to pass an additional `isIPv6` boolean?

Reply:
I think it is ok to break this! It is pretty recent + will break at compile time.

@braginini what do you think?